go/interpreter/lfvm/instructions_test.go

```diff
@@ -870,3 +873,184 @@ func TestExpansionCostOverflow(t *testing.T) {
 		}
 	}
 }
+
+func TestSStoreGasCost(t *testing.T) {
+
+	tests := map[string]struct {
+		runtimeSetup   func(*tosca.MockRunContext)
+		revision       tosca.Revision
+		gas            tosca.Gas
+		expectedStatus Status
+	}{
+		"berlin-enough-gas": {
```

Comment:
nit: use spaces or underscores

(go test names are going to replace spaces with underscores, and using `-` will end up with names like 
`test/bla_bla-bla`)